Header header
float64[9] homography
float64    depth
bool 	   valid
